toptal/picasso
# feat(Accordion): [FX-1092] rework borders

[FX-1092](https://toptal-core.atlassian.net/browse/FX-1092)

### Description

Change default Accordion borders and add more options

### How to test

- Navigate to storybook https://picasso.toptal.net/fx-1092-accordion-style/iframe.html?id=components--accordion&7af4e8108af4b60e02526805d72c35b357519aae=&viewMode=story
- Make sure all examples are consistent with the design requirements

### Screenshots

![F66F18F8-5194-495F-AE98-2DC150E07838](https://user-images.githubusercontent.com/254110/99712847-4547b180-2aac-11eb-9b92-c45d72239356.GIF)

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>
